src/translations/strings.yaml

```diff
@@ -146,15 +146,10 @@ splittunnel:
       - Success! You added “%1” to this list.
     comment:
       - Displayed in notification after the user added a missing app to the split-tunnel List; %1 is the name of the added application
-  infoText2:
-    value:
-      - For any questions about setting app exclusions, we’ve provided some helpful tips on our website.
-    comment:
-      - This displays as informational text below split tunneling settings. The link directs to a sumo article which will help certain users who are having network issues using this feature.
   infoLinkText: Learn more
   infoCardDescription:
-    value: VPN must be off to edit App exclusions
-    comment: Displays inside the information card at the top of the app exclusions screen when the VPN is on
+    value: VPN must be off to edit Excluded apps
```

Comment:
To change an existing string you need a new ID.

Reply:
I wrote it that way initially. But do you think the translations for "App exclusions" and "Excluded apps" will be substantially different? If not, perhaps we don't need to bother the translators with this one. I'll defer to you.

Reply:
In general, if we think a fix it's worth for English (for consistency), we also want it in other locales. The translations will be definitely different between the two.

Reply:
Sounds good, I've updated the PR to use new string IDs.